include/boost/geometry/algorithms/detail/disjoint/multirange_geometry.hpp

```diff
@@ -40,9 +40,9 @@ class unary_disjoint_geometry_to_query_geometry
     {}
 
     template <typename QueryGeometry>
-    inline bool apply(QueryGeometry const& query_geometry) const
+    inline bool operator()(QueryGeometry const& query_geometry) const
     {
-        return BinaryPredicate::apply(query_geometry, m_geometry, m_strategy);
+        return ! BinaryPredicate::apply(query_geometry, m_geometry, m_strategy);
```

Comment:
Again this is now `intersects` predicate. Assuming that `BinaryPredicate` is `disjoint` predicate because technically this predicate is negating whatever `BinaryPredicate` ism but that's another issue. So the name should be changed as well.

Reply:
Regarding the names. Note that if we decided to name these predicate as `intersects` then we'd end up with confusing types, i.e. disjoint predicates passed into intersects predicates, e.g.:
`unary_intersects_geometry_to_query_geometry` taking `disjoint_range_segment_or_box`. So I guess it'd be better to use name containing e.g. `unary_not`, `unary_not_disjoint` etc.